tsconfig.json

```diff
@@ -3,6 +3,8 @@
   "compilerOptions": {
     "outDir": "dist/esm",
     "lib": ["ESNext", "DOM"],
+    "verbatimModuleSyntax": false,
```

Comment:
Can you explain why we need this? And is it a workaround need to be fixed later, or should it be kept in future versions? So does the next line.

Reply:
This is a workaround before @chatie/tsconfig updates. ```verbatimModuleSyntax``` is the replacement of ```importsNotUsedAsValues``` after typescript 5.0. See: https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues

Reply:
If it's a workaround, can you file an issue to record it and put the issue link in the comment so we can track it in the future? Please also do this in similar changes because it's important for future maintenance. Thanks!

Reply:
https://github.com/wechaty/wechaty/issues/2551